unittests/test_utility.py

```diff
@@ -280,6 +280,22 @@ def test_load_unknown_path(self):
             self.assertEqual('foo', e.name)
             self.assertEqual('/foo', e.path)
 
+    def test_load_dir_starts_with_ddot(self):
+        parent_dir = os.path.basename(os.path.abspath(os.pardir))
+        filename = '../%s/reframe' % parent_dir
+        module = util.import_module_from_file(filename)
+        self.assertEqual(reframe.VERSION, module.VERSION)
+        self.assertEqual('reframe', module.__name__)
+        self.assertIs(module, sys.modules.get('reframe'))
+
+    def test_load_file_starts_with_ddot(self):
+        parent_dir = os.path.basename(os.path.abspath(os.pardir))
```

Comment:
Better use `..` instead of `os.pardir` to be clearer and consistent with other usages.

Reply:
Also, I wouldn't write the test this way. I'd instead change temporarily to a directory and do a relative import from there. This is the faulty scenario in any case. You can use the `utility.os_ext.change_dir` context manager for doing the directory change safely.